lib/spreewald/web_steps.rb

```diff
@@ -659,20 +668,20 @@
   end
 end.overridable
 
-# Tests that an input, button or checkbox with the given label is disabled.
-Then /^the "([^\"]*)" (field|button|checkbox) should( not)? be disabled$/ do |label, kind, negate|
+# Tests that an input, button, checkbox or radio button with the given label is disabled.
+Then /^the "([^\"]*)" (field|button|checkbox|radio button) should( not)? be disabled$/ do |label, kind, negate|
   if Spreewald::Comparison.compare_versions(Capybara::VERSION, :<, "2.1")
-    if kind == 'field' || kind == 'checkbox'
-      element = find_field(label)
-    else
+    if kind == 'button'
       element = find_button(label)
+    else
+      element = find_field(label)
     end
     expect(element[:disabled]).send(negate ? :to : :not_to, eq(nil))
   else
```

Comment:
I'd expect that you could use `find_with_disabled` for this step as well, as it should work in any Capybara version.